projects/plugins/jetpack/modules/contact-form/grunion-response-email-template.php

```diff
@@ -0,0 +1,277 @@
+												</td>
+											</tr>
+											<tr>
+												<td align="left" class="footer-november-2022-app-links" style="border-collapse: collapse;">
+													<a href="https://play.google.com/store/apps/details?id=com.jetpack.android" style="color: #00AADC; text-decoration: none; width: 115px; display: inline-block; margin-bottom: 64px;"><img src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/play-store-badge.png" class="footer-november-2022-app-image" width="115" height="33" style="line-height: 100%%;outline: none; text-decoration: none; border: 0 none; display: inline-block; height: 33px;"></a>
+
+													<a href="https://apps.apple.com/us/app/jetpack-website-builder/id1565481562" style="color: #00AADC; text-decoration: none; width: 115px; display: inline-block; margin-bottom: 64px;"><img src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/app-store-badge.png" class="footer-november-2022-app-image" width="115" height="33" style="line-height: 100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block; height: 33px;"></a>
+												</td>
+											</tr>
+											<tr>
+												<td align="left" class="footer-november-2022-socials-section" style="border-collapse: collapse;">
+													<a href="https://twitter.com/jetpack" style="color: #00AADC; text-decoration: none; display: inline-block; padding-right: 22px;"><img aria-label="jetpack twitter page" src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/twitter-dark.png" width="20" height="20" style="height: auto; line-height: 100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block;"></a>
+
+													<a href="https://www.facebook.com/jetpackme" style="color: #00AADC; text-decoration: none; display: inline-block; padding-right:22px;"><img aria-label="jetpack facebook page" src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/facebook-dark.png" width="20" height="20" style="height: auto; line-height:100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block;"></a>
+
+													<a href="https://www.linkedin.com/company/jetpack-for-wordpress/" style="color: #00AADC; text-decoration: none; display: inline-block; padding-right: 22px;"><img aria-label="jetpack linkedin page" src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/linkedin-dark.png" width="20" height="20" style="height: auto; line-height:100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block;"></a>
+												</td>
+											</tr>
+											<tr>
+												<td align="left" style="border-collapse: collapse;">
+													<p class="footer-november-2022-address-text" style="direction: ltr; color: #101517; font-size: 12px; line-height: 1.5; margin-bottom: 10px; margin-top: 24px;">
```

Comment:
We have a lot of hardcoded `ltr` and `left` alignments in here. Should we make those dynamic, using `is_rtl` maybe?

Reply:
Done.